src/command_modules/azure-cli-container/azure/cli/command_modules/container/_params.py

```diff
@@ -69,6 +69,7 @@ def load_arguments(self, _):
         c.argument('environment_variables', nargs='+', options_list=['--environment-variables', '-e'], type=_environment_variables_type, help='A list of environment variable for the container. Space-separated values in \'key=value\' format.')
         c.argument('secure_environment_variables', nargs='+', type=_secure_environment_variables_type, help='A list of secure environment variable for the container. Space-separated values in \'key=value\' format.')
         c.argument('secrets', secrets_type)
+        c.argument('network_profile', help='The network profile id')
```

Comment:
This should support name or ID. 
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#supporting-name-or-id-parameters

Reply:
The networking team's sdk is not updated with the network profile to query and get the ID. I will do a follow up PR when they release the new SDK but we wanted to get this out to unblock us just in case.

Reply:
You don't need the Network SDK to construct the ID. Please see the referenced example.

Reply:
networkProfile is a top-level resource, so you should only need the name, resource group and subscription.

`/subscriptions/xxxx/resourceGroups/tjp-rg/providers/Microsoft.Network/networkProfiles/profile1`

Reply:
I misunderstood. I thought it was a sub resource. I'll make the change now.